c-example/runtime_loading.c

```diff
@@ -0,0 +1,90 @@
+#include "bridgestan.h"
+#include <stdio.h>
+
+#ifdef _WIN32
+// hacky way to get dlopen and friends on Windows
+
+#include <libloaderapi.h>
+#include <errhandlingapi.h>
+#define dlopen(lib, flags) LoadLibraryA(lib)
+#define dlsym(handle, sym) (void*)GetProcAddress(handle, sym)
+
+char* dlerror() {
```

Comment:
Honestly, I don't know how important this is, and my guess is not that important, but it came to mind so I'm saying something.  Should the `dlerror` signature be `void dlerror(char* err)`, so as to free memory outside of the function?

Reply:
This only exists to mirror the `dlerror` that is built-in on Linux/Mac, so we're constrained to the same signature. It definitely does leak memory, but since all we do in this tiny example is call it right before quitting the program, it doesn't matter for us really

Reply:
Yep, that makes and sense and is fair.  I was just (probably overly) worried about our example being taken too literally.

Reply:
Yeah, hopefully someone doing this "for real" on Windows would try a bit harder... my comment in the code calling this hacky was mostly out of this same concern